docs/docsource/installation/upgrade/upgrade_zenko.rst

```diff
@@ -101,40 +101,75 @@ To upgrade from 1.0.x to 1.1:
 
      $ helm upgrade {{zenko-release-name}} ./zenko --set cosmos.enabled=false
 
-   If you are using custom values, be sure to reuse the options.yaml file on
-   upgrades.
+   If you are using custom values, be sure to reuse your options.yaml file on
+   any upgrade.
    ::
 
       $ helm upgrade {{zenko-server-name}} ./zenko --set cosmos.enabled=false -f options.yaml
 
-#. After the upgrade has stabilized all pod rollouts and prior functionality
-   is verified, run the following command to finalize the MongoDB compatibility
+#. After the upgrade has stabilized and all pod rollouts and prior functionality
```

Comment:
not sure why this sentence is changed? It made more sense in the older version

Reply:
Thanks! Looks like I misunderstood what was being said and edited accordingly.